### PR TITLE
Dedicated server documentation mtt 1905

### PR DESCRIPTION
**Pull request by s-omeilia-unity**
**Select the type of change:**

- [x] Small Changes - Typos, formatting, slight revisions
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site and Docusaurus

**Purpose of the Pull Request:**
Adding more content around dedicated servers. (In-progress)
Involved moving the terminology and game architecture pages to a more visible location.

**Issue Number:** 
[MTT-1905](https://jira.unity3d.com/browse/MTT-1905)

### CODE REVIEW

**Comment by johann-r-unity** [comment]
I don't see the content I provided you with.  Is it supposed to be in another section entirely ?

**Comment by s-omeilia-unity** [comment]
> 

The steps for the Unity Dedicated Server module under that header. Some of the content is blended in context above then the rest will make sense under the Unity Dedicate Server module header.

**Comment by s-omeilia-unity** [comment]
@JesseOlmer and @SamuelBellomo This is the PR mentioned in the SoS meeting. 

This is the section of content I'm reworking while simultaneously adding more to the Dedicated Server information with info on the Unity Dedicated Server module: https://docs-multiplayer.unity3d.com/netcode/current/learn/multiplayer-game-arhitecture (Introduction, Network Topologies, Listen server and host architecture).

I can push this to sandbox if it's easier to discuss what this PR looks like versus the live content now.

Semi-related, this is the conversation and info regarding the Relay situation referenced between Sam and I during the SoS as well: https://unity.slack.com/archives/G01J4H2C405/p1651675565693329 Haven't gotten a response on PM on this.

Just reiterating from SoS, typical doc stance is to not include non-Unity options in documentation if there is a Unity option available. That's a docs standard typically across every company I've worked for. How we want to handle that as multiplayer we should define and then I'll game plan how to rectify the rest of our docs for consistency.

File: docs/reference/dedicated-server.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+description: NEW DESCRIPTION
+---
+
+## What is a dedicated server?
+
+A dedicated server is a standalone, headless platform build optimized for running in the cloud as a game server application. They simulate game worlds without supporting direct input or output, except that required for their administration. Players must connect to the server with separate client programs to see and interact with the game.
+
+![Dedicated Server](/img/ded_server.png)
+
+Many multiplayer games rely on a client-host (a listen server setup) as both the client and server. Because it relies on the host machine's internet connection and hardware capabilities, its simplistic setup can cause performance issues such as lag, slow loading times, crashes, loss of save files, and dropped games. A dedicated server is often a more stable and enjoyable gaming experience.
```
Comment (johann-r-unity) [inline docs/reference/dedicated-server.md:14]: There is also the problematic of unfair advantage and cheat concerns.

File: docs/reference/dedicated-server.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+A dedicated server manages all hosting duties, such as: player locations and activity, games rules, physics, weapons, and equipment. However, the dedicated server does not run the game engine or render anything. These functions (such as graphics, game controls, and sound) is responsible on the client's machine. The dedicated server _instructs_ the client machines on what to render and when.
```
Comment (johann-r-unity) [inline docs/reference/dedicated-server.md:16]: For the weapons and the equipment's, I would say it depends of your game.

File: docs/reference/dedicated-server.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+* Facilitates a reliable, stable, and faster connection
+* Often eliminates lag, crashes, or glitches
+* Can handle high traffic
+* Less prone to corrupt save files
+* Higher levels of security based on admin login definitions and the server's sole purpose to run the game
+
+## Disadvantages
+
+* Setup is more complex than other network topologies
+* Requires regular maintenance of both the hardware and software of the server device
```
Comment (johann-r-unity) [inline docs/reference/dedicated-server.md:30]: The hardware side maintenance is not really  a concern if the server is hosted in the cloud. Which tends to be more and more the case.

File: docs/reference/dedicated-server.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+* Greater processing, memory, and storage capabilities relative to other hosting solutions
+* Facilitates a reliable, stable, and faster connection
+* Often eliminates lag, crashes, or glitches
+* Can handle high traffic
+* Less prone to corrupt save files
+* Higher levels of security based on admin login definitions and the server's sole purpose to run the game
+
+## Disadvantages
+
+* Setup is more complex than other network topologies
```
Comment (johann-r-unity) [inline docs/reference/dedicated-server.md:29]: The complexity is not that much related to the network topology. I don't think that is that much different from a host in this regard. 
Complexity is mostly about the difficulty to manage two distinct projects : 1 for the player, 1 for the server.

**Comment by johann-r-unity** [comment]
> > 
> 
> The steps for the Unity Dedicated Server module under that header. Some of the content is blended in context above then the rest will make sense under the Unity Dedicate Server module header.

Saw it, thank you!

**Comment by armstrongl** [comment]
Closed in favor of https://github.com/Unity-Technologies/com.unity.multiplayer.docs/pull/873

**Pull request closed without merging**